### PR TITLE
ssp_v2_changes

### PR DESCRIPTION
**Pull request by ioigoume**

### CODE REVIEW

File: roles/ssp/tasks/configure-common.yml [hunk 1]
```diff
@@ -227,7 +227,24 @@
...
+  template:
+    src: "{{ item }}"
+    dest: "{{ ssp_configdir }}/module_metarefresh.php"
+    backup: yes
+    force: yes
+  with_first_found:
+    - "config/module_metarefresh-{{ ssp_major_version }}.{{ ssp_minor_version }}.php.j2"
+    - "config/module_metarefresh-{{ ssp_major_version }}.php.j2"
+    - "config/module_metarefresh.php.j2"
+  when: "'metarefresh' in ssp_mods_enabled and {{ ssp_major_version|float > 2}}"
```
Comment (NicolasLiampotis) [inline roles/ssp/tasks/configure-common.yml:247]: Similar to the previous comment, could you try:

```
when: ssp_major_version | int >= 2
```

File: roles/ssp/tasks/configure-common.yml [hunk 2]
```diff
@@ -227,7 +227,24 @@
     - "config/config-metarefresh-{{ ssp_major_version }}.{{ ssp_minor_version }}.php.j2"
     - "config/config-metarefresh-{{ ssp_major_version }}.php.j2"
     - "config/config-metarefresh.php.j2"
-  when: "'metarefresh' in ssp_mods_enabled"
+  when: "'metarefresh' in ssp_mods_enabled and {{ ssp_major_version|float < 2}}"
```
Comment (NicolasLiampotis) [inline roles/ssp/tasks/configure-common.yml:230]: This type of condition causes the following warning:
```
[WARNING]: conditional statements should not include jinja2 templating
delimiters such as {{ }} or {% %}. Found: {{ ssp_major_version|float < 2}}
```
Could you try this instead?

```
when: ssp_major_version | int < 2
```

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: roles/ssp/tasks/configure-common.yml (hunk 2 above)
Comment (ioigoume) [reply roles/ssp/tasks/configure-common.yml]: done

File: roles/ssp/templates/config/config-2.php.j2 [hunk 3]
```diff
@@ -566,11 +566,11 @@ $config = [
...
-        'core' => true,
-        'admin' => true,
-        'saml' => true,
-        'cron' => true,
-        'metarefresh' => true,
+        {% if ssp_mods_enabled is defined %}
+            {% for mod in ssp_mods_enabled %}
+        '{{ mod }}' => true,
+            {% endfor %}
+        {% endif %}
```
Comment (NicolasLiampotis) [inline roles/ssp/templates/config/config-2.php.j2:573]: Add admin to ssp_mods_enabled in "ssp/defaults/main.yml"

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**